nutaudet/Target.cxx

```diff
@@ -385,16 +388,30 @@ void Target::ConstructGeometry()
   TGeoVolume *volBrick = new TGeoVolume("Brick",Brick,air);
   volBrick->SetLineColor(kCyan);
   volBrick->SetTransparency(1);   
+
+  TGeoBBox *Lead, *Tungsten; //need to separate the two cases
+  TGeoVolume *volLead, *volTungsten;
     
-  TGeoBBox *Lead = new TGeoBBox("Pb", EmulsionX/2, EmulsionY/2, LeadThickness/2);
-  TGeoVolume *volLead = new TGeoVolume("Lead",Lead,lead);
-  volLead->SetTransparency(1);
-  volLead->SetLineColor(kGray);
-  //volLead->SetField(magField2);
+  if (fDesign < 4){  
+   Lead = new TGeoBBox("Pb", EmulsionX/2, EmulsionY/2, LeadThickness/2);
+   volLead = new TGeoVolume("Lead",Lead,lead);
+   volLead->SetTransparency(1);
+   volLead->SetLineColor(kGray);
+   //volLead->SetField(magField2);
+  }
+  else{
+   Tungsten = new TGeoBBox("W", EmulsionX/2, EmulsionY/2, LeadThickness/2);
+   volTungsten = new TGeoVolume("Tungsten",Tungsten,tungsten);
+   volTungsten->SetTransparency(1);
+   volTungsten->SetLineColor(kGray);
+   //volLead->SetField(magField2);
+
+  }
     
   for(Int_t n=0; n<NPlates; n++)
     {
-      volBrick->AddNode(volLead, n, new TGeoTranslation(0,0,-BrickZ/2+BrickPackageZ/2+ EmPlateWidth + LeadThickness/2 + n*AllPlateWidth)); //LEAD
+      if (fDesign < 4) volBrick->AddNode(volLead, n, new TGeoTranslation(0,0,-BrickZ/2+BrickPackageZ/2+ EmPlateWidth + LeadThickness/2 + n*AllPlateWidth)); //LEAD
+      else volBrick->AddNode(volTungsten, n, new TGeoTranslation(0,0,-BrickZ/2+BrickPackageZ/2+ EmPlateWidth + LeadThickness/2 + n*AllPlateWidth)); //LEAD
```

Comment:
The comment here should probably be `TUNGSTEN`?

Reply:
Yes, sorry my mistake